.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.idea
```

Comment:
This is not needed

Reply:
I agree. It's IDE specifics so we don't need it.